lib/rucio/core/lock.py

```diff
@@ -206,6 +211,10 @@ def get_files_and_replica_locks_of_dataset(scope, name, nowait=False, restrict_r
             filter(models.DataIdentifierAssociation.scope == scope,
                    models.DataIdentifierAssociation.name == name)
 
+        if total_threads and total_threads > 1:
```

Comment:
total_threads twice?

Reply:
yes, this will not use sharding for tt values : None (the default), zero (serendipitously) and one (intentionally)